tests/test_osmnx.py

```diff
@@ -141,6 +142,23 @@ def test_stats() -> None:
     G_clean = ox.consolidate_intersections(G, rebuild_graph=False)
 
 
+def test_extract_edge_bearings_directionality() -> None:
```

Comment:
These unit tests can be folded into the `test_bearings` function.

Reply:
I'm not seeing a `test_bearings` test. I do see `test_stats` but that test is already doing a lot.

Reply:
I thought I had already named one that but apparently it was just in my mental to-do list and I forgot :) Nevermind!